### PR TITLE
Milestone-1,Hello Microverse 

### PR DESCRIPTION
**Pull request by nati2323**
## Hello Microverse Project!
 - added html file 
 - added css file
 - added test file
 -.........................................
 ## NELSON's REVIEW
- make your comments a little bit more descriptive
- ## Jorge's Review
- Use imperative mode in commit messages.
- ## REJOICE Review
- use what camel case for naming your classes and commit messages.
- avoid keywords like color for naming your classes. thanks.

### CODE REVIEW

**Comment by nati2323** [comment]
#Milestone-1,Hello Microverse
 - added Html files
- added Css files
- added test files

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+Prerequisites
+ .Use browser
+ .Install
+Authors
+ .GitHub: @nati2323
+Give a ⭐️ if you like this project!
+License
+ .This project is MIT licensed.
+
+
```
Comment (lily-coder) [inline README.md]: - [ ] [REQUIRED] Great job in adding the README file, however, as stated in the project requirements, please use the [Microverse README Template](https://github.com/microverseinc/readme-template/blob/master/README.md) but remember to customize it to your own project. If you don't know how you can add the titles and all that, when you visit the page given, click on raw and copy the template but remember to customize it to your own project.

- You can add the sections such as in the **Authors** you can add your twitter and linkedin link, also sections that apply such as **Contributing**, and **Show your support** 😊😊

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @nati2323 , 

Good job so far!

To Highlight! 🟢
✔️ Great job with the linters

There are some issues that you still need to work on to go to the next project but you are almost there!

image


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [REQUIRED] Since you have already made commit messages, please consider writing your commit messages in the future in a professional way, by writing them capitalized and in the imperative tense. For example, instead of _fixed indentation error_ , You can write it as _Fix indentation error_. You can visit [This Page](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to see how you can write your commit messages more professionally

- [ ] [REQUIRED] Please also consider making your pull request title more descriptive. Instead of _`project-1 content`_ you can have it as _`Set up a "Hello Microverse" project`_ (this is just an example). You can follow [this page](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information.

- [ ] [REQUIRED] Please also consider making your branch the name of an action for example `Added footer`

- [ ] [REQUIRED] As stated in the project requirements, please add a  `A test.md file with the text "This file should be ignored by git".`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nati2323 — COMMENTED** [review]
I did the correction! Thanks

File: README.md (hunk 1 above)
Comment (nati2323) [reply README.md]: I did the correction. Thanks!

**Pull request closed without merging**